tools/get_number_fxa_users.js

```diff
@@ -0,0 +1,38 @@
+var conf = require('../loop/config').conf;
+var redis = require("redis");
+
+var storage = conf.get("storage");
+
+function main(callback) {
+  if (storage.engine === "redis") {
+    var options = storage.settings;
+    var client = redis.createClient(
+      options.port,
+      options.host,
+      options.options
+    );
+    if (options.db) client.select(options.db);
+
+    var multi = client.multi();
+    multi.keys("userid.*");
```

Comment:
We have a lua command that can count this on server side: https://github.com/mozilla-services/loop-server/blob/master/tools/get_average_calls_per_user.js#L17